frontend/app/src/components/dialogs/WatcherDialog.vue

```diff
@@ -166,12 +166,20 @@
 <script lang="ts">
 import cloneDeep from 'lodash/cloneDeep';
 import { Component, Emit, Prop, Vue, Watch } from 'vue-property-decorator';
+import { mapActions } from 'vuex';
 import MessageDialog from '@/components/dialogs/MessageDialog.vue';
-import { Watcher } from '@/services/types-api';
-import { OpTypes, WatcherType, WatcherTypes } from '@/services/types-common';
+import {
+  OpTypes,
+  Watcher,
+  WatcherType,
+  WatcherTypes
+} from '@/services/defi/types';
 
 @Component({
-  components: { MessageDialog }
+  components: { MessageDialog },
```

Comment:
I think we can remove this as we don't use the MessageDialog component anymore. I must have forgotten to remove it when we finished the watcher component for 1.5.0.